vendor/lib/lib_path.cmd

```diff
@@ -85,13 +85,13 @@ exit /b
 
     if /i "%position%" == "append" (
       if "!found!" == "0" (
-        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:";%find_query%\"$"
+        echo "%path%"|%WINDIR%\System32\findstr >nul /I /R /C:";%find_query%\"
         call :set_found
```

Comment:
The `$` at the end is part of a regex that says find this at the end of the string removing it removes the intention of this entire line of code.